lib/src/view/user/user_profile.dart

```diff
@@ -47,11 +56,45 @@ class UserProfile extends StatelessWidget {
               child: userFullName,
             ),
           if (user.profile?.bio != null)
-            Text(
-              user.profile!.bio!,
+            Linkify(
+              onOpen: (link) async {
+                if (link.originText.startsWith('@')) {
+                  final userId =
```

Comment:
It is better to immediately go to the user screen with the id, and load the user while in the screen.